api-reference/40 Common Types/AIIntegration/AIIntegration.md

```diff
@@ -0,0 +1,60 @@
+---
+id: AIIntegration
+module: common/ai-integration
+export: AIIntegration
+---
+---
+##### shortDescription
+A type of the main AI integration class.
+
+---
+You must create an instance of this class by passing in an object of the [AIProvider](/Documentation/ApiReference/Common_Types/AIProvider/) type. This instance should then be passed to any components that support AI features.
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    const aiIntegration = new DevExpress.aiIntegration(provider);
+
+    $("#htmlEditor").dxHtmlEditor({
+        // ...
+        aiIntegration,
+    });
+
+##### Angular
+
+    <!-- tab: app.component.ts -->
+    import { AIIntegration } from 'devextreme-angular/ai-integration';
+    // ...
+    export class AppComponent {
+        ai = new AIIntegration(provider);
```

Comment:
It's better not to use “ai”, I think. It might confuse users. Don't you think so?